sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/ScriptTransformationSuite.scala

```diff
@@ -228,6 +228,26 @@ class ScriptTransformationSuite extends SparkPlanTest with SQLTestUtils with Tes
         'e.cast("string")).collect())
     }
   }
+
+  test("SPARK-30973: ScriptTransformationExec should wait for the termination of script") {
+    (0 until 10).foreach { _ =>
+      assume(TestUtils.testCommandAvailable("/bin/bash"))
```

Comment:
nit: `assume(!Utils.isWindows && TestUtils.testCommandAvailable("/bin/bash"))`

Reply:
I find other UT cases do not have this condition, so I write the same code here.